### PR TITLE
Remove module attributes

### PR DESCRIPTION
**Pull request by drwestco**
See if removing type="module" allows older browsers in.

### CODE REVIEW

File: client/toDataByLocation.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 export default function toDataByLocation(data) {
   const headers = data.values[1];
-  const approvedIndex = headers.findIndex( e => e === 'approved' );
-  const stateIndex = headers.findIndex( e => e === 'state' );
-  const cityIndex = headers.findIndex( e => e === 'city' );
-  const latIndex = headers.findIndex( e => e === 'lat' );
-  const lngIndex = headers.findIndex( e => e === 'lng' );
+  const approvedIndex = headers.indexOf('approved');
```
Comment (awong-dev) [inline client/toDataByLocation.js:3]: Shouldn't babel handle this?

Comment (drwestco) [reply client/toDataByLocation.js:3]: I'd think so. Still trying to decipher the babel docs regarding polyfills.

Comment (drwestco) [reply client/toDataByLocation.js:3]: Think I got it sorted. Will verify with good ol' IE in the morning.

Comment (awong-dev) [reply client/toDataByLocation.js:3]: you rock. Thank you!!!

Comment (drwestco) [reply client/toDataByLocation.js:3]: Turning on the polyfills via Babel does the trick. It really bloats the .entry.js files, though:

image

File: views/stats.handlebars [hunk 2]
```diff
@@ -17,8 +17,8 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.i18n/1.0.7/jquery.i18n.language.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.i18n/1.0.7/jquery.i18n.messagestore.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.i18n/1.0.7/jquery.i18n.parser.min.js"></script>
-  <script type="module" src="generated/i18n.entry.js?v=v{{ version }}"></script>
-  <script type="module" src="generated/stats.entry.js?v=v{{ version }}"></script>
+  <script src="generated/i18n.entry.js?v=v{{ version }}"></script>
```
Comment (nburt) [inline views/stats.handlebars:20]: fixing this in a separate pr but public paths need a `/` at the start of the path

**Pull request closed without merging**